### PR TITLE
pin version of roiextractors

### PR DESCRIPTION
**Pull request by h-mayorquin**
Testing pinning exact version of roi extractors

### CODE REVIEW

File: requirements-full.txt [hunk 1]
```diff
@@ -14,9 +14,9 @@ psutil==5.8.0
 lxml==4.6.5
 opencv-python==4.5.1.48
 spikeextractors==0.9.9
-spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git
+spikeinterface @ git+https://github.com/SpikeInterface/spikeinterface.git@12b7863684b705e3d0ae8165bb3009143c70530c
 neo==0.10.2
-roiextractors @ git+https://github.com/catalystneuro/roiextractors.git
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@10265be4a1eaece7461296a2877cc11c84bf87dc
```
Comment (bendichter) [inline requirements-full.txt:19]: it would be better to cut a release on pypi and pin to that

Comment (CodyCBakerPhD) [reply requirements-full.txt:19]: For ROIExtractors we can control that of course, but not for SpikeInterface

Comment (CodyCBakerPhD) [reply requirements-full.txt:19]: We're mostly debugging the terrible MacOS+Py3.8 combination of git installation right now

Comment (bendichter) [reply requirements-full.txt:19]: what's terrible about it?

Comment (CodyCBakerPhD) [reply requirements-full.txt:19]: Producing unexpected behavior never seen before unique to that combination of os and py version, we think we may have pinned it down

Comment (CodyCBakerPhD) [reply requirements-full.txt:19]: Can confirm fixed, all Mac CI are past that installation point now

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Thanks @h-mayorquin 

This PR

- cleans up numpy float casting deprecation warnings coming from roiextractors
- also reveals behavior of `pip install -e .[full]` occurring after initial `pip install -e .` to not overwrite package versions, will think on how best to deal with that
- will cut a new release of roiextractors soon, looking for other cleanup warnings before then

**Pull request merged**